change/@ni-nimble-components-df093e6a-7cba-46ed-8d5d-07670af90719.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think you could link #1293 to this PR. Or, possibly just close it if it is just a duplicate work item.